src/components/Footer.js

```diff
@@ -1,16 +1,47 @@
-import React from 'react'
+import React from 'react';
+import { FaFacebookSquare, FaTwitterSquare, FaLinkedin, FaEnvelope } from 'react-icons/fa';
 
 function Footer() {
     return (
         <footer className="page-footer font-small">
+            {/* Social Media */}
+            <div className="social-media">
+                <a href="https://m.facebook.com/devconzm/">
+                    <FaFacebookSquare 
+                        size={50}
+                    />
+                </a>
+                <a href="https://twitter.com/devcon_zm">
+                    <FaTwitterSquare 
+                        size={50}
+                    />
+                </a>
+                <a href="https://www.linkedin.com/company/developer-conference-zambia-19">
+                    <FaLinkedin 
+                        size={50}
+                    />
+                </a>
+                <a href="mailto:info@devcon.co.zm">
+                    <FaEnvelope 
+                        size={50}
+                    />
+                </a>
+                <div>
+                    <h6>Check us out on social media!</h6>
+                </div>
+            </div>
+            {/* Social Media */}
 
             {/* Copyright */}
-            <div className="container footer-copyright text-center py-3">&copy; <a href="http://devcon.co.zm">2019 DevCon Zambia.</a> All rights reserved.
+            <div className="container footer-copyright text-center py-3">
+                <p>&copy;2019 <a href="http://devcon.co.zm"> DevCon Zambia</a> all rights reserved.</p>
```

Comment:
The a on all rights needs to be capitalized

Reply:
Just commited and changed the "A" back to uppercase, sorry about that.